### PR TITLE
Fix: edit kid photo dropdown

### PR DESCRIPTION
**Pull request by Judithlourde**
One Line Description
Edit kid photo dropdown style

Requirements
- Expectation: Please refer to [Figma ](https://www.figma.com/design/qUBlZE5TIgpqShKTVcSvRo/Kids-First-WITT(Desktop)?node-id=14512-130900&t=seuHdaHYobfLzmRZ-4)design.
- NOTE : There is no Figma design for effect of mousing over dropdown items. Check with designers on expectation.

Notes
There is a Figma design illustrating the effect of mousing over dropdown items, which can be found here:
https://www.figma.com/proto/qUBlZE5TIgpqShKTVcSvRo/Kids-First-WITT(Desktop)?node-id=14512-130520&node-type=frame&t=iItelqg7RezdCKWy-0&scaling=scale-down&content-scaling=fixed&page-id=0%3A1&starting-point-node-id=14909%3A60377

Test Steps
1. Open the application. 
2. Navigate to the kid's info page
3. Check Edit Kid Photo dropdown design 

UI/UX Outcome
image

Checklist
- [x] Pull Request title includes the issue number and a brief description of the change.
- [x] All changes should be tested and verified to work correctly.
- [x] Code follows frontend best practices.
- [x] Branch names follow the conventions in the contributing file.
- [x] Commit messages follow the naming conventions in the contributing file.

### CODE REVIEW

File: src/components/Dashboard/Kids/KidsInfo/Dropdown/PhotoOptionsDropdown.module.css [hunk 1]
```diff
@@ -13,17 +13,39 @@
...
 }
 
 .dropdownItem {
-  padding: 12px 16px;
+  padding: 0.4em 0.8em; 
+  width: 200px;
   cursor: pointer;
+  display: flex;
+  justify-content: center;
 }
```
Comment (nandini92) [inline src/components/Dashboard/Kids/KidsInfo/Dropdown/PhotoOptionsDropdown.module.css:47]: Please also review the font sizing and alignment ([Figma](https://www.figma.com/design/qUBlZE5TIgpqShKTVcSvRo/Kids-First-WITT(Desktop)?node-id=13353-54946&t=l5ZrRszaMUTsFk4z-4)).
image
image

**Review by nandini92 — CHANGES_REQUESTED** [review]
Great job Judith!  All that is remaining is some text style changes.

File: src/components/Dashboard/Kids/KidsInfo/Dropdown/PhotoOptionsDropdown.module.css (hunk 1 above)
Comment (Judithlourde) [reply src/components/Dashboard/Kids/KidsInfo/Dropdown/PhotoOptionsDropdown.module.css:47]: Thank you @nandini92 for letting me know. I’ve addressed the changes you requested. Let me know if there’s anything else!
Snag_2661830e

**Review by nandini92 — APPROVED** [review]
Great job @Judithlourde ! This PR is approved!

**Review by pandjyoti — APPROVED** [review]

**Pull request merged**